src/components/SignoffSummary/index.jsx

```diff
@@ -38,9 +46,14 @@ function SignoffSummary(props) {
           const key = `${role}-${index}`;
 
           return (
-            <ListItem key={key} dense className={classes.requiresSignoffsList}>
+            <ListItem key={key} className={classes.signoffsList}>
               <ListItemText
-                primary={`${count} member${count > 1 ? 's' : ''} of ${role}`}
+                primary={
+                  <Typography>{`${count} member${
+                    count > 1 ? 's' : ''
+                  } of ${role}`}</Typography>
+                }
+                className={classes.listItemText}
```

Comment:
Looks a bit off with the Typography component since the variant is changed:

#### With Typogpraphy

![Screen Shot 2019-07-23 at 10 26 37 AM](https://user-images.githubusercontent.com/3766511/61720237-653e5800-ad34-11e9-81b5-bb019f036da6.png)


#### Without Typography

![Screen Shot 2019-07-23 at 10 26 20 AM](https://user-images.githubusercontent.com/3766511/61720239-653e5800-ad34-11e9-9d29-cade67f2ba7b.png)

`ListItemText` internally uses a Typography component to render the primary prop.

Reply:
Hm, it looks a little different for me. With the Typography, I see:
![image](https://user-images.githubusercontent.com/49649/61743489-6b97f880-ad63-11e9-9c5d-3a8db7a72f0f.png)

Reply:
Interesting. Let's try the same thing but written differently. Can you try:

```diff
diff --git a/src/components/SignoffSummary/index.jsx b/src/components/SignoffSummary/index.jsx
index ccb9b7b..998734f 100644
--- a/src/components/SignoffSummary/index.jsx
+++ b/src/components/SignoffSummary/index.jsx
@@ -1,4 +1,4 @@
-import React, { Fragment } from 'react';
+import React from 'react';
 import { object } from 'prop-types';
 import classNames from 'classnames';
 import { makeStyles } from '@material-ui/styles';
@@ -26,6 +26,10 @@ const useStyles = makeStyles(theme => ({
     paddingTop: 0,
     paddingBottom: 0,
   },
+  signedBy: {
+    display: 'flex',
+    alignItems: 'center',
+  },
 }));
 
 function SignoffSummary(props) {
@@ -48,11 +52,7 @@ function SignoffSummary(props) {
           return (
             <ListItem key={key} className={classes.signoffsList}>
               <ListItemText
-                primary={
-                  <Typography>{`${count} member${
-                    count > 1 ? 's' : ''
-                  } of ${role}`}</Typography>
-                }
+                primary={`${count} member${count > 1 ? 's' : ''} of ${role}`}
                 className={classes.listItemText}
               />
             </ListItem>
@@ -71,14 +71,21 @@ function SignoffSummary(props) {
             {listOfSignoffs.map(([username, signoffRole]) => (
               <ListItemText
                 key={username}
+                disableTypography
                 primary={
-                  <Fragment>
+                  <Typography
+                    component="p"
+                    className={classes.signedBy}
+                    variant="body2">
                     {username}
-                    {' - '}
-                    <Typography color="textSecondary" variant="caption">
+                    &nbsp; - &nbsp;
+                    <Typography
+                      component="span"
+                      color="textSecondary"
+                      variant="caption">
                       {signoffRole}
                     </Typography>
-                  </Fragment>
+                  </Typography>
                 }
                 className={classes.listItemText}
               />
```

Reply:
That changes the padding ever so slightly, but still looks good here:
![image](https://user-images.githubusercontent.com/49649/61745566-d0555200-ad67-11e9-8d1d-c5f98793800c.png)

Reply:
Looks good to me.